### PR TITLE
Get rid of "About this guidance" (index.html)

### PR DESCRIPTION
**Pull request by davidread**
I've basically deleted the "About this guidance" page, which appears on the front page of the guidance.

I want to do this because the front page is rather wasted on just this bit of contact info. You have to navigate to see nearly anything. It's much better to describe the product here, so people coming know what it's all about, with a similar idea as https://www.notifications.service.gov.uk/

Front page before:
Screen Shot 2020-05-20 at 17 37 11

After:
Screen Shot 2020-05-20 at 17 36 45

The "contact" links are moved to the footer, and look like this:
Screen Shot 2020-05-20 at 17 32 52

### CODE REVIEW

File: config/tech-docs.yml [hunk 1]
```diff
@@ -37,3 +37,8 @@ github_repo: moj-analytical-services/user-guidance
 # Slack options
 owner_slack_workspace: asdslack
 default_owner_slack: '#analytical_platform'
+
+footer_links:
+  Get in touch by email: mailto:analytical_platform@digital.justice.gov.uk
+  Slack: https://asdslack.slack.com/messages/C4PF7QAJZ/
+  Report an issue with these docs in GitHub: https://github.com/moj-analytical-services/platform_user_guidance/issues
```
Comment (calumabarnett) [inline config/tech-docs.yml:44]: I'm not sure we need this one as there are 'Report issue' links on each page anyway. The URL also refers to the old guidance.

File: config/tech-docs.yml [hunk 2]
```diff
@@ -37,3 +37,8 @@ github_repo: moj-analytical-services/user-guidance
 # Slack options
 owner_slack_workspace: asdslack
 default_owner_slack: '#analytical_platform'
+
+footer_links:
+  Get in touch by email: mailto:analytical_platform@digital.justice.gov.uk
+  Slack: https://asdslack.slack.com/messages/C4PF7QAJZ/
```
Comment (calumabarnett) [inline config/tech-docs.yml:43]: Could we change this to 'Get in touch by Slack' for consistency?

File: source/index.html.md.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Analytical Platform User Guidance
-weight: 10
+title: Introduction
+weight: 20
 ---
 
-<%= partial 'documentation/index' %>
+<%= partial 'documentation/introduction' %>
```
Comment (calumabarnett) [inline source/index.html.md.erb:6]: Could we change the partial from `introduction.md` to `index.md` for consistency?

File: config/tech-docs.yml (hunk 1 above)
Comment (davidread) [reply config/tech-docs.yml]: 👍 done

File: config/tech-docs.yml (hunk 2 above)
Comment (davidread) [reply config/tech-docs.yml]: 👍 done

File: source/index.html.md.erb (hunk 3 above)
Comment (davidread) [reply source/index.html.md.erb]: 👍 done

**Pull request merged**